.vscode/launch.json

```diff
@@ -12,7 +12,7 @@
             "smartStep": true,
             "sourceMaps": true,
             "outFiles": ["${workspaceFolder}/out/**/*", "!${workspaceFolder}/**/node_modules**/*"],
-            "preLaunchTask": "Compile",
+            // "preLaunchTask": "Compile",

```

Comment:
I assume you don't want to submit this.

Reply:
First Ian, now you... why can't i check in crap...

Reply:
(pun intended)

Reply:
Time for me to create a new launch config